Add common errors to faq

jacoballen1 opened the pull request:
This PR is currently a work in progress, feel free to suggest errors to add.

jacoballen1 committed:
FAQ expansion

Added headers to common errors

hugmanrique commented:
Nice work! Two minor things:
Can you replace the absolute URLs by relative ones? Otherwise, when the site goes live these will need to be changed.
The last error (Velocity forwarding mode) contains a typo (`connecty`)

jacoballen1 commented:
@hugmanrique I will ultimately put the "correct" url in (the one that will be used in the release). But thanks for the tips!

hugmanrique commented:
There's no need for that, e.g. replace `https://newsite-staging.velocitypowered.com/wiki/users/forwarding/` by `/wiki/users/forwarding/`

A248 commented on docs/users/faq.md:
```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
+``
+
+``
+Can't conenct to server lobby: Your server did not send a forwarding request to the proxy. 
```
connect*

jacoballen1 commented:
Fixed it for both links

A248 commented on docs/users/faq.md:
```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
+``
+
+This error is a result of only enabling forwarding on the backend server. To fix this error, 
+ensure that you have set up the correct player information forwarding method on the proxy.
```
> correct player information forwarding method

This sounds a bit wordy.

jacoballen1 replied:
Perhaps "correct forwarding method"?

A248 replied:
I think "ensure you have set up player info(rmation) forwarding on the proxy" would be sufficient and concise.

jacoballen1 replied:
well, at that rate we should just leave it how it is... no?
It's not like we have a ton going on for this part of the documentation so we can get away with a little more precision.

astei commented on docs/users/faq.md:
```diff
@@ -19,4 +19,30 @@ However, certain plugins may have Velocity ports available or alternatives
+
+# Common Errors
+
+``
```
Let's add some headers?

jacoballen1 replied:
I have added some headers in the most recent commit. Let me know if that is what you were looking for.

jacoballen1 commented:
I have rebased the branch onto the master to be up to date and condensed the commits into one. I also removed the `package-lock.json` that boosted this pr up to 20k lines.
